tests/api/entities/resolver/resolve.test.js

```diff
@@ -398,6 +398,18 @@ describe('entities:resolve:on-labels', () => {
   })
 })
 
+describe('entities:resolve:on-external-terms', () => {
+  // Fragile test: its validity depends on the stability of Wikipedia and Wikidata
+  xit('should resolve the author when a work label appears in Wikipedia', async function () {
+    this.timeout(60000)
+    // The author is in Wikidata, but not his work (this test will become useless once that work is in Wikidata)
+    const workLabel = "Mémoires d'outre-espace"
+    const authorLabel = 'Enki Bilal'
```

Comment:
can it throw if workLabel is found in author's work, then we can safely unxit this test?

Reply:
@jum-s my mistake, the test I intended to write was not finished: 87116c6

Reply:
but its still a xitted test, while we could programmatically check if bilal have a wdEntitiy called "Memoire d'outre espace"

Reply:
I gave it a try: a5a002b8